packages/sui-react-initial-props/README.md

```diff
@@ -79,8 +79,10 @@ contextFactory(
   match({routes, location: req.url}, function (error, redirectLocation, renderProps = false) {
     const Target = (props) => <div>{props.fromInitialProps}</div>
     ssrComponentWithInitialProps({ context, renderProps, Target })
-      .then(({ initialProps, reactString }) => {
+      .then(({ initialProps, reactString, performance }) => {
         // you have here the initialProps retrieved from the page in case you need it
+        console.log('Time spended resolving the getInitialProps method', performance.getInitialProps)
```

Comment:
`Time spent resolving the getInitialProps method`

Reply:
ok